config/settings.defaults.coffee

```diff
@@ -37,23 +37,12 @@ module.exports =
 				docsWithHistoryOps: ({project_id}) -> "DocsWithHistoryOps:{#{project_id}}"
 
 		project_history:
-			port: process.env["HISTORY_REDIS_PORT"] or process.env["REDIS_PORT"] or "6379"
-			host: process.env["HISTORY_REDIS_HOST"] or process.env["REDIS_HOST"] or "localhost"
-			password: process.env["HISTORY_REDIS_PASSWORD"] or process.env["REDIS_PASSWORD"] or ""
-			maxRetriesPerRequest: parseInt(process.env['REDIS_MAX_RETRIES_PER_REQUEST'] or "20")
-			key_schema:
-				projectHistoryOps: ({project_id}) -> "ProjectHistory:Ops:{#{project_id}}"
-				projectHistoryFirstOpTimestamp: ({project_id}) -> "ProjectHistory:FirstOpTimestamp:{#{project_id}}"
-
-		new_project_history:
 			port: process.env["NEW_HISTORY_REDIS_PORT"] or "6379"
 			host: process.env["NEW_HISTORY_REDIS_HOST"]
```

Comment:
Should there be a fallback to REDIS_HOST for the dev environment?

Reply:
Yes, I'll add a fallback option now, to be consistent with all the other redis configurations.  I had deliberately left out a default while doing the migration as I didn't want to accidentally send data to the wrong redis (better for it to crash by failing to connect to localhost).